### ISSUE TITLE
Are center loss and ArcFace loss useful?

### ISSUE THREAD

**Issue by FlyuZ** [damo-cv/TransReID#34]
I found that there are center loss and ArcFace loss codes in the code, but they don’t seem to be used. I can’t find the results of comparative experiments in the paper. Are they useful?
Thank you！

**Comment by FlyuZ** [comment]
And Circle Loss？

**Comment by michuanhaohao** [comment]
We haven't tried these loss functions. You can try them. You may need to adjust the training parameters.
